src/game/engine/tracker.cpp

```diff
@@ -14,12 +14,78 @@
  *            LICENSE
  */
 #include "tracker.h"
+#include "house.h"
+#include "team.h"
+#include "teamtype.h"
+#include "unit.h"
+#include "vessel.h"
+#include "aircraft.h"
+#include "building.h"
+#include "bullet.h"
+#include "infantry.h"
+#include "anim.h"
+#include "iomap.h"
+#include "logic.h"
+#include "vortex.h"
+#include "trigger.h"
+#include "triggertype.h"
 
-void Detach_This_From_All(int32_t target, int unk)
+void Detach_This_From_All(target_t target, BOOL a2)
 {
-    // TODO Needs most of the top level object classes.
 #ifdef GAME_DLL
```

Comment:
If this is complete, remove the ifdefs and hook and test it so its used in all builds.